shell/platform/android/io/flutter/embedding/engine/FlutterJNI.java

```diff
@@ -106,6 +106,13 @@ public static native void nativeInit(
       @NonNull String engineCachesPath,
       long initTimeMillis);
 
+  /**
+   * Create the default font manager provided by SkFontMgr::RefDefault() which is a process-wide
+   * singleton owned by Skia. Note that, the first call to SkFontMgr::RefDefault() will take
+   * noticeable time, but later calls will return a reference to the preexisting font manager.
+   */
+  public static native void nativeCreateDefaultFontManager();
```

Comment:
Change this to `nativePrefetchDefaultFontManager` (and do the same for the `CreateDefaultFontManager` function in `flutter_main.cc`).

I'd prefer to avoid the name `Create` given that the function does not return a font manager instance.

Reply:
done